integration/vite-css-dev-test.ts

```diff
@@ -300,8 +332,28 @@ test.describe("Vite CSS dev", () => {
         UPDATED_TEST_PADDING_VALUE
       );
 
+      // Ensure CSS updates were handled by HMR
       await expect(input).toHaveValue("stateful");
 
+      // The following change triggers a full page reload, so we check it after the HMR changes
```

Comment:
Had a bit of trouble understanding this comment on my first read, so took a shot at clarifying:

```suggestion
      // The following change triggers a full page reload, so we check it after all the checks for HMR state preservation
```

Feel free to tweak it more!